eng/scripts/Get-SdkVersion.ps1

```diff
@@ -0,0 +1,6 @@
+# Obtain Current SDK Version
+. ${PSScriptRoot}\SdkVersion-Common.ps1
+
+$match = $SdkVersionFile -match $VersionStringRegEx
+$SdkVersion = $Matches[2]
```

Comment:
Probably worth doing a named group here so we don't accidently change the regex and miss updating things like this. It might actually be worth just adding a function in the common script next to the regex to get the version.